xbmc/view/GUIViewState.cpp

```diff
@@ -395,9 +395,9 @@ bool CGUIViewState::AutoPlayNextItem()
   return false;
 }
 
-bool CGUIViewState::JumpToFirstUnplayedItem()
+int CGUIViewState::SelectFirstUnplayedItem()
```

Comment:
drop this and the declaration after you moved the selection code to CGUIWindowVideoNav